rd-net/Lifetimes/Lifetimes.csproj

```diff
@@ -46,6 +46,10 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="JetBrains.NETFramework.ReferenceAssemblies.net35" Version="1.0.1">
```

Comment:
Should we conditionally include it only for `net35`?

Reply:
No, because this package has condition inside.